mantidproject/mantid
# Hb3a parse binary 23124

Enable LoadSpiceXML2DDet to load binary file from HB3A.


**To test:**

Check unit test with a sample binary file.  In the instrument view, powder ring can be observed.

Fixes #23124 .


*This does not require release notes* because **it only affects HB3A reduction**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
